packages/neo-one-monitor/src/MonitorBase.js

```diff
@@ -780,6 +792,73 @@ export default class MonitorBase implements Span {
     }
   }
 
+  report(report: Report): void {
+    report.logs.forEach(log => {
+      const newLog = {
+        name: log.name,
+        level: log.level,
+        message: log.message,
+        labels: log.labels,
+        data: log.data,
+        error: undefined,
+      };
+      const { error } = log;
+      if (error != null) {
+        newLog.error = new Error(error.message);
+        if (error.stack != null) {
+          newLog.error.stack = error.stack;
+        }
+      }
+      this._logger.log(newLog);
+    });
+
+    for (const counterName of Object.keys(report.metrics.counters)) {
+      const counterMetric = report.metrics.counters[counterName];
+      const counter = this.getCounter({
+        name: counterMetric.metric.name,
+        help: counterMetric.metric.help,
+        labelNames: counterMetric.metric.labelNames,
+        directReport: true,
+      });
+
+      counterMetric.values.forEach(value => {
```

Comment:
This is a good example of where using `forEach` makes sense - you're doing a side effect (`.inc`) and as such, the iteration should represent that.